ts/screens/profile/PreferencesScreen.tsx

```diff
@@ -210,7 +212,14 @@ class PreferencesScreen extends React.Component<Props, State> {
               }
               onPress={this.handleEmailOnPress}
             />
-
+            {// Check if spid phone number exists
```

Comment:
Could you put it below emails (email and SPID email)?
Now the mobile phone is in the between